### PR TITLE
Create a ChipThemer for material and outlined schemes

### PR DESCRIPTION
**Pull request by jmdetloff**
Before:
simulator screen shot - iphone x - 2018-04-27 at 13 43 06

After:
simulator screen shot - iphone x - 2018-04-27 at 13 42 03

### CODE REVIEW

File: components/Chips/src/ChipThemer/MDCChipViewThemer.h [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ limitations under the License.
+ */
+
+#import "MDCChipViewScheme.h"
+
+#import "MaterialChips.h"
+
+#import <Foundation/Foundation.h>
+
+/** Applies material style to MDCChipView objects. */
```
Comment (jverkoey) [inline components/Chips/src/ChipThemer/MDCChipViewThemer.h]: Update these API docs using the most recent version of MDCButtonThemer as a template please.

File: components/Chips/src/ChipThemer/MDCChipViewScheme.h [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ limitations under the License.
+ */
+
+#import "MaterialChips.h"
+#import "MaterialColorScheme.h"
+#import "MaterialTypographyScheme.h"
+
+#import <Foundation/Foundation.h>
+
+/** Defines a readonly immutable interface for chip style data to be applied by a themer. */
```
Comment (jverkoey) [inline components/Chips/src/ChipThemer/MDCChipViewScheme.h]: Update these docs with MDCButtonScheming as a template please.

File: components/Chips/src/ColorThemer/MDCChipViewColorThemer.h [hunk 3]
```diff
@@ -34,12 +34,13 @@
...
 
 /**
- Applies a color scheme's properties to a stroked MDCChipView.
+ Applies a color scheme's properties to a stroked @c MDCChipView.
 
  @param colorScheme The color scheme to apply to the component instance.
- @param strokedChipView A component instance to which the color scheme should be applied.
+ @param chipView @c A component instance to which the color scheme should be applied.
  */
-+ (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme
```
Comment (jverkoey) [inline components/Chips/src/ColorThemer/MDCChipViewColorThemer.h]: I believe this API already landed unfortunately, so we'll have to mark it as to-be-deprecated.

**Review by jverkoey — DISMISSED** [review]

File: components/Chips/src/ColorThemer/MDCChipViewColorThemer.h (hunk 3 above)
Comment (jmdetloff) [reply components/Chips/src/ColorThemer/MDCChipViewColorThemer.h]: It has landed, but it seems no one is using it yet. With that in mind can we sneak it out without deprecating?

Comment (jverkoey) [reply components/Chips/src/ColorThemer/MDCChipViewColorThemer.h]: You can mark it as deprecated. Once we cut the next release you can delete it.

**Review by jverkoey — APPROVED** [review]

**Pull request merged**